ipfx/dataset/hbg_nwb_data.py

```diff
@@ -0,0 +1,45 @@
+from typing import Dict, Any, List, Optional
+import abc
+
+from ipfx.stimulus import StimulusOntology
+from ipfx.dataset.ephys_nwb_data import EphysNWBData, get_finite_or_none
+
+
+class HBGNWBData(EphysNWBData):
+    """
+    Provides an Ephys Data Interface to an HBG generated NWB file
+
+    """
+
+    def __init__(self,
+                 nwb_file: str,
+                 ontology: StimulusOntology,
```

Comment:
I would rather pass the str stimulus_ontology file name not the StimulusOntology object. This look cleaner to me since this parallels the other argument (nwb_file). What are we gaining by passing the ontology object?